docs/src/main/asciidoc/_includes/devtools/create-app.adoc

```diff
@@ -1,7 +1,7 @@
 [role="primary asciidoc-tabs-sync-cli"]
 .CLI
 ****
-[source,bash,subs=attributes+]
+[source, bash, subs=attributes+]
```

Comment:
Where does this rule come from?

From what I can see, we don't put spaces here in our doc and the AsciiDoctor doc doesn't either.
So apart from requiring additional work for everyone, I don't see the value of it so an explanation would be welcome?

Reply:
@gsmet Hello. The main purpose was to have it unified so that it is easier for me to search for these headers and some lines of code below them, and replace them with different lines of code. To have it automated, I needed to have it consistent first.

Such as, for example, if i would be doing this in IDEA:

**Regex:**
((\*) .*
\+)?
\[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"\]
\.pom.xml
----
((.*\n)+?)----
(\+?)
\[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"\]
\.build.gradle
----
((.*\n)+?)----


**Replacement:**
$1
$2* Using Maven:
+
--
[source,xml]
----
$3----
--
+
$2* Using Gradle:
$5
--
[source,gradle]
----
$6----
--


I am not a coder and not a regex expert, so I asked my friend @jmartisk if he could help me with composition of a bash script so that all writers can use it, not matter what IDE or Linux they use.
Problem is, the original regexs that works fine withing IDEa, doesnt work in bash, and some tweak and limitations needs to be addressed.
Also, my motivation here is not to put more burden on your shoulders @gsmet . I wanted to take this issue completely out of Quarkus , and using the script to patch the RHBQ files before each release, or when needed..

To make the coding easier, I want to reach some level of consistency at least in these files (which are completely fine and written in a super smart way, but i need to apply some cosmetics to workaround lacking and abandoned dowstream publication system) that are put in includes.